definitions/hello-world/hello-world.yaml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
I wouldn't expect to see this file in a PR for adding a yaml linter. This should be in a different PR or completely removed as we don't really need this anymore.

Reply:
Same comment above.